mmv1/third_party/terraform/services/gkeonprem/resource_gkeonprem_bare_metal_cluster_test.go.erb

```diff
@@ -381,6 +393,12 @@ func testAccGkeonpremBareMetalCluster_bareMetalClusterUpdateManualLb(context map
         }
       }
     }
+    binary_authorization {
+      evaluation_mode = "DISABLED"
+    }
+    upgrade_policy {
+      policy = "SERIAL"
```

Comment:
Can you please change the value to `CONCURRENT` to test the update of this field from `SERIAL`?

Reply:
Updated. Thanks for checking!